examples/nextjs/src/app/flow/Controls.tsx

```diff
@@ -1,113 +1,129 @@
-import { type ChangeEvent, useState } from 'react';
-import { useFlow } from '@speechmatics/flow-client-react';
-import { useAudioDevices } from '@speechmatics/browser-audio-input-react';
+'use client';
+import { type FormEventHandler, useCallback, useMemo, useState } from 'react';
+import {
+  type AgentAudioEvent,
+  useFlow,
+  useFlowEventListener,
+} from '@speechmatics/flow-client-react';
+import { MicrophoneSelect } from '@/lib/components/MicrophoneSelect';
+import { PersonaSelect } from './PersonaSelect';
+import { getJWT } from '../actions';
+import {
+  usePcmAudioListener,
+  usePcmAudioRecorder,
```

Comment:
One thing I'd like to bring to the group is some naming conventions, and think airbnb's rules are a good guidance for this [https://github.com/airbnb/javascript?tab=readme-ov-file#naming--Acronyms-and-Initialisms](https://github.com/airbnb/javascript?tab=readme-ov-file#naming--Acronyms-and-Initialisms)
What do you think? So Acronyms would be capitalised etc

Reply:
I agree we should have standards for this. Just to note in this case, both `usePcmAudioListener/Recorder` are coming from our `browser-audio-input-react` package, which exports them at the top level, so changing it there would mean a breaking change.

Reply:
ok cool. Maybe a ticket for this change in a general way for later then?